conf.py

```diff
@@ -107,12 +107,24 @@
 
 html_copy_source = False
 
-# Meta tags for SEO
-html_meta = {
+# Meta tags for SEO and social sharing
+myst_html_meta = {
```

Comment:
Note: this will only work with MyST, so it will only work with Markdown files, and some of our documentation is still in RST.

For example, the [main page](https://docs.epics-controls.org/en/latest/) is still in RST, and wouldn't have these meta elements.

Reply:
Ah good point.